Microsoft.ML.FairLearn/api-example.cs

```diff
@@ -0,0 +1,52 @@
+// See https://aka.ms/new-console-template for more information
+using Microsoft.ML;
+using Microsoft.ML.AutoML;
```

Comment:
I'm not entirely sure where AutoML comes in here, but I'd be cautious about advocating for automatically "fixing" fairness issues. It will optimize for some metrics, but the context really matters here:
- How was the metric chosen?
- What is the resulting model? How did it manage to "reduce disparity?" Is that way appropriate in this application context?
- What is the dataset? Why is there disparity to begin with? Where does it originate? Is it something we can address in a meaningful way using ML or is this a problem better addressed in another way (e.g., policy)?
- ...

@riedgar-ms and I have discussed this dozens of times and so far stayed away from AutoML for fairness. I'm sure there are responsible use cases, but it takes a significant amount of time to dive into all the questions (starting with some of the ones I wrote above) and the outcome may just be that we don't want to use such a model. [I should add that coming to the conclusion to not go forward with a model is perfectly valid and sometimes/often the best choice!]

Now I probably totally overinterpreted what you're trying to do with this `api-example.cs` file, but I thought it's nevertheless useful to share some of the thought process we typically apply.